src/libopensc/cards.h

```diff
@@ -231,6 +231,12 @@ enum {
 
 	/* JPKI cards */
 	SC_CARD_TYPE_JPKI_BASE = 31000,
+
+	/* CAC cards */
+	SC_CARD_TYPE_CAC_BASE = 31000,
```

Comment:
Thanks. I missed this during the rebase to the master (with already a new card). Fixed in 56c164986